src/test-apps/happy/topologies/standalone/thread_wifi_on_tap_ap_service.sh

```diff
@@ -22,7 +22,7 @@
 # - one thread-wifi border_gateway device BorderRouter
 # - one access-point router onhub
 # - one service node instance cloud
-
+export PATH=$PATH:$TRAVIS_BUILD_DIR/src/test-apps/happy/bin
```

Comment:
`$TRAVIS_BUILD_DIR` now makes this specific to the Travis CI integration. please use something more general

Reply:
done